frontend/packages/insights-plugin/OWNERS

```diff
@@ -0,0 +1,8 @@
+reviewers:
+  - bond95
+  - tisnik
+approvers:
+  - bond95
+  - tisnik
+labels:
+  - component/insights
```

Comment:
You'll need a PR to the openshift/release repo to add this label. See https://github.com/openshift/release/pull/4526

Reply:
I created PR for that https://github.com/openshift/release/pull/12398